sdk/core/core-amqp/src/util/utils.ts

```diff
@@ -28,6 +29,29 @@ export interface AsyncLockOptions {
    */
   Promise?: any;
 }
+
+/**
+ * Options to configure the channelling of the AMQP connection over Web Sockets.
+ */
+export interface WebSocketOptions {
+  /**
+   * @property
+   * The WebSocket constructor used to create an AMQP connection over a WebSocket.
+   * This option should be provided in the below scenarios:
+   * - The TCP port 5671 which is that is used by the AMQP connection to Event Hubs is blocked in your environment.
```

Comment:
Since this is now in core-amqp, "Event Hubs" should not be mentioned.
Skip it altogether

cc @HarshaNalluru, @chradek

Reply:
Welcome back @ramya-rao-a !